runbot_travis2docker/tests/test_runbot_build.py

```diff
@@ -78,6 +78,38 @@ def test_20_jobs_pr(self):
         _logger = logging.getLogger(__name__ + '.def test_20_jobs_pr')
         self.run_jobs('refs/pull/1')
 
+    def test_30_ssh_server(self):
+        self.repo.update()
+        self.repo.killall()
```

Comment:
These tests are too much expensive
Maybe we could re-use the original method to add custom validations for ssh keys
(I know we are breaking tests' standards)